### DISCUSSION TITLE
wakatime api

### DISCUSSION THREAD

**Discussion by Devzjta** [aulianza/aulianza.id#296] [General]
How do I get the three API keys from Wakatime?

I only found the Secret API Key at https://wakatime.com/settings/account

How do I get all three?

WAKATIME_CLIENT_ID=
WAKATIME_CLIENT_SECRET=
WAKATIME_CLIENT_REFRESH_TOKEN=

**Comment by KevinASaputra** [comment]
how to get wakatime refresh token??

**Comment by aulianza** [comment]
Hi @Devzjta @KevinASaputra sorry for late responses, currently use WAKATIME_API_KEY instead, please see my latest code, thank you
